courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -701,95 +587,76 @@ Implementation-Dependent Types
 Unconstrained Formal Parameters
 ---------------------------------
 
-* Take bounds from actual parameters
+* Take bounds from **actual** parameters
+* Also applies to :ada:`return` values
```

Comment:
No - return object is bounded by the function (caller typically uses the function to initialize an unconstrained object)

Reply:
Return as in formal return, which can be declared unconstrained, rewriting that